Testreglar/4.1.2/Nett/nett-4.1.2c.json

```diff
@@ -0,0 +1,133 @@
+{
+	"namn": "Nett-4.1.2c Iframe har et tilgjengelig navn som beskriver formålet med innholdet i iframe.",
+	"id": "nett-4.1.2c",
+	"testlabId": 437,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Der det er brukt iframe-elementet i koden, har iframe et tilgjengelig navn, som beskriver formålet med innholdet i iframe.</p>",
```

Comment:
Gjelder dette kravet for hidden iframe element og? Feks googel analytics? I act-regel cae76 Iframe element has non-empty accessible name brukes den følgende : 

> This rule applies to iframe elements that are [included in the accessibility tree](https://act-rules.github.io/rules/cae760#included-in-the-accessibility-tree) and that can be accessed by [sequential focus navigation](https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation).

Reply:
En skjult iframe er ikke omfattet av kravet, fordi det er ikke en del av innholdet som brukerne oppfatter som et enkeltkontrollelement med en bestemt funksjon ([brukergrensesnittkomponent](https://www.w3.org/Translations/WCAG21-no/#dfn-user-interface-components)). 
Skjulte elementer kan skapet tastaturfelle, jeg tror at det er derfor ACT-rules bruker "iframe elements that are [included in the accessibility tree](https://act-rules.github.io/rules/cae760#included-in-the-accessibility-tree) and that can be accessed by [sequential focus navigation](https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation)"

Vi har oppdatert steg 2.2.